ark/utils/spatial_analysis_utils.py

```diff
@@ -172,12 +187,11 @@ def compute_close_cell_num(dist_mat, dist_lim, num, analysis_type,
     # iterating k from [j, end] cuts out 1/2 the steps (while symmetric)
     for j, m1n in enumerate(mark1_num):
         for k, m2n in enumerate(mark1_num[j:], j):
-            close_num[j, k] = np.sum(
-                dist_mat_bin[np.ix_(
-                    np.asarray(mark1poslabels[j] - 1, dtype='int'),
-                    np.asarray(mark1poslabels[k] - 1, dtype='int')
-                )]
-            )
+            close_num[j, k] = np.sum(dist_mat_bin.loc[
```

Comment:
I think breaking this up into steps will make code readability clearer. First get the sum, then assign the sum